packages/cli/__tests__/zosfiles/__system__/create/pds/cli.files.create.dataset.pds.system.test.ts

```diff
@@ -199,7 +199,8 @@ describe("Create Partitioned Data Set", () => {
         it("should fail creating a partitioned data set extended due to invalid data-set-type", () => {
             const response = runCliScript(__dirname + "/__scripts__/command/command_create_pdse_fail_dsntype.sh",
                 TEST_ENVIRONMENT, [user]);
-            expect(response.stderr.toString()).toContain("Invalid zos-files create command 'dsntype' option: NONLIBRARY");
+            expect(response.stderr.toString()).toContain("Unable to perform this operation due to the following problem.");
+            expect(response.stderr.toString()).toContain("NONLIBRARY");
```

Comment:
I see that the system test was changed after `dsntype` validation was modified - since we are keeping the validation as-is now, should the changes in this test be reverted?